internal/cloudapi/server.go

```diff
@@ -298,7 +298,7 @@ func (server *Server) ComposeStatus(w http.ResponseWriter, r *http.Request, id s
 			Status: composeStatusFromJobStatus(status, &result),
 			UploadStatus: &UploadStatus{
 				Status: result.UploadStatus,
-				Type:   "aws",
```

Comment:
@Gundersanne this won't be a problem for image-builder, I assume? This value is currently ignored?

Reply:
This is fine yea, it's currently ignored as you say.